src/Tests/SettingsDataTrait.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\acquia_lift\Tests\SettingsDataTrait.
+ */
+
+namespace Drupal\acquia_lift\Tests;
```

Comment:
The duplication was intentional because unit tests and functional tests should not share data fixtures. Some major reasonings are:
* Unit tests and functional tests have different focus, so their fixture data will naturally rest at different setups and complexity.
* Fixtures are to be expanded and updated later as the tests grow, it is a good idea to limit the impact to the same type of tests in a project of this size. In the future we may even create multiple sets of fixtures to serve type of tests.

Otherwise, to eliminate duplication is as simple as changing the require_once()'s path.